src/mock/ytelse/foreldrepenger-mock.ts

```diff
@@ -0,0 +1,66 @@
+import * as faker from 'faker/locale/nb_NO';
+import * as moment from 'moment';
+
+import navfaker from 'nav-faker/dist/index';
+
+import {
+    Foreldrepengerperiode,
+    ForeldrepengerResponse,
+    Foreldrepengerrettighet
+} from '../../models/ytelse/foreldrepenger';
+import { getPeriode } from '../person/periodeMock';
+import { fyllRandomListe } from '../utils/mock-utils';
+import { getHistoriskUtbetaling, getKommendeUtbetaling } from './ytelse-utbetalinger-mock';
+
+export function getForeldrepenger(fødselsnummer: string): ForeldrepengerResponse {
+    faker.seed(Number(fødselsnummer));
+    navfaker.seed(fødselsnummer);
+
+    return {
+        foreldrepenger: getForeldrepengerrettighet(fødselsnummer)
+    };
+}
+
+function getForeldrepengerrettighet(fødselsnummer: string): Foreldrepengerrettighet {
+    return {
+        forelder: fødselsnummer,
+        andreForeldersFnr: navfaker.fødselsnummer.generer(),
+        antallBarn: navfaker.random.number(5),
+        barnetsFødselsdato: moment(faker.date.recent()).format(moment.ISO_8601.__momentBuiltinFormatBrand),
+        dekningsgrad: navfaker.random.number(95),
+        fedrekvoteTom: moment(faker.date.recent()).format(moment.ISO_8601.__momentBuiltinFormatBrand),
+        mødrekvoteTom: moment(faker.date.recent()).format(moment.ISO_8601.__momentBuiltinFormatBrand),
+        foreldrepengetype: 'FTYPE',
+        graderingsdager: navfaker.random.number(100),
+        restDager: navfaker.random.number(50),
+        rettighetFom: moment(faker.date.recent()).format(moment.ISO_8601.__momentBuiltinFormatBrand),
```

Comment:
Kanskje vi skal legge til noe slikt i mocken som enten:
1. Returner et moment objekt
2. Returnerer et date-objekt
3. Returner en datoen som en string parset i ISO_8601.

I dette tilfellet er det jo kanskje 3 som er best? Men kanskje litt for spesifikt? Det holder kanskje å gi en dato, så er det opp til brukeren selv å bestemme formatering etc..